test/compress/arrays.js

```diff
@@ -8,7 +8,7 @@ holes_and_undefined: {
     expect: {
         w=[1,,];
         x=[1,2,void 0];
-        y=[1,,2];
+        y=[1,,2,];
```

Comment:
Even though v8 has good support I prefer to have a ref spec. Will change if needed though. This is just making things a little more complicated.